Server/src/main/java/Database/TheAuPair/Controllers/FuelCostController.java

```diff
@@ -0,0 +1,46 @@
+package Database.TheAuPair.Controllers;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import static java.time.temporal.ChronoUnit.SECONDS;
+
+@RestController
+@CrossOrigin(origins = "*")
+public class FuelCostController {
+  @Value("${fuelsa.api-key}")
+  private String fuelSAApiKey;
+
+  @GetMapping("/getCurrentFuelPrices")
+  public String getCurrentFuelPrices() throws URISyntaxException, IOException, InterruptedException {
```

Comment:
Can those exceptions not be changed to just be Exception instead of 3 separate ones?